java/src/main/java/com/google/appengine/tools/mapreduce/impl/MapReduceConstants.java

```diff
@@ -32,7 +32,7 @@ private MapReduceConstants() {}
    */
   public static final int DEFAULT_IO_BUFFER_SIZE = 1 * 1024 * 1024;
 
-  public static final int GCS_IO_BLOCK_SIZE = 256 * 1024 * 1024; // 256KB
+  public static final int GCS_IO_BLOCK_SIZE = 256 * 1024; // 256KB
```

Comment:
🤦  

now as to why that causes the sort test to make 37 reduce calls rather than the 100 expected by the test - 🤷  my guess we would be some implementation artifact of the FW coupled to some bug, where basically 2/3rds (rounded) of the input gets lost.  and in CI only 1/4th (rounded) of input gets lost (hence the 76) ... but yeah, weird.  vry unclear to me (and concerning) why changing block size would affect it one way or the other - one would assume that FW just uses this value to allocate objects in GCS, and files the unneeded portion with empty chars or something.

but whatever. seems to work now in local.